### PR TITLE
LPS-159766 compare against localized key instead of 'advanced' literal

### PR DESCRIPTION
**Pull request by jgarcialr**

### CODE REVIEW

**Comment by jgarcialr** [comment]
ci:test:sf

**Comment by jgarcialr** [comment]
ci:test:relevant

File: modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/utils/objectFields.js [hunk 1]
```diff
@@ -106,7 +106,8 @@ export function getSelectedValue(value) {
 }
 
 export function getObjectFieldName({settingsContext}) {
-	const getAdvancedColumn = ({title}) => title.toLowerCase() === 'advanced';
+	const getAdvancedColumn = ({title}) =>
+		title.toLowerCase() === Liferay.Language.get('advanced').toLowerCase();
 	const fieldsFromAdvancedColumn = settingsContext.pages.find(
 		getAdvancedColumn
 	)?.rows[0].columns[0].fields;
```
Comment (carolmariaabb) [inline modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/utils/objectFields.js:113]: Hi @jgarcialr,

I think we could simplify this code by doing something like:

```
const fieldsFromAdvancedColumn = settingsContext.pages.find(
	({title}) => title === Liferay.Language.get('advanced')
)?.rows[0].columns[0].fields;
```

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

File: modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/utils/objectFields.js (hunk 1 above)
Comment (jgarcialr) [reply modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/utils/objectFields.js]: Hi @carolmariaabb,

I'll add this change, thanks for the review :smile:

**Review by carolmariaabb — APPROVED** [review]

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**